kodev

```diff
@@ -145,116 +145,116 @@ ${SUPPORTED_TARGETS}"
         PARAM=$(echo "${1}" | awk -F= '{print $1}')
         VALUE=$(echo "${1}" | awk -F= '{print $2}')
         case "${PARAM}" in
-            -v | --verbose)
```

Comment:
@NiLuJe This seems to be a new default; it can be reverted to how it was with `-ci`. Any opinions? The rationale is that it's the same as if elif else for all intents and purposes and should therefore be indented the same.

See https://github.com/mvdan/sh/commit/2e429b0d9b620c0cc360d0adba08f6818fe7c0cf and the associated issue for more details.

Reply:
I personally prefer the indented style, but that's basically what it is: a stylistic choice ;).

Reply:
(Hell, I actually personally align the `;;` with the conditional, and not the branch ;).)

Reply:
TL;DR: Neither are really my own preference, so, no strong opinion on this ;).

Reply:
Guess I'll just keep it at the default then. Google does it the other way, after all. :-P

Reply:
(Although, the indented style is closer to mine, so I'll have less of a chance to shout at shfmt for getting it wrong, which I'll most likely will ^^).

Reply:
lol, I was just about to click merge

Reply:
Sorry ^^.